### PR TITLE
Version 0.17.1

### PR DESCRIPTION
**Pull request by LarsStegman**
Changes: [0.17.0..master](https://github.com/encode/starlette/compare/0.17.0...master)
GitHub Draft release: [here](https://github.com/encode/starlette/releases/tag/untagged-3c955988c265f4781516)

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.17.1
+
+November 17, 2021
+
+### Fixed
+* Fix `IndexError` in authentication `requires` when wrapped function arguments are distributed between `*args` and `**kwargs` [#1335](https://github.com/encode/starlette/pull/1355)
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
* Fix `IndexError` in authentication `requires` when wrapped function arguments are distributed between `*args` and `**kwargs` [#1335](https://github.com/encode/starlette/pull/1335).
```

**Review by Kludex — CHANGES_REQUESTED** [review]
Next steps:
- @tomchristie needs to give you maintainer status, 
- Create the draft release.
- Get this approved.
- Create tag 0.17.1 and push it
- Draft -> Release

**Review by Kludex — APPROVED** [review]

**Comment by lovelydinosaur** [comment]
Invite sent.

As @kludex says, once you're a maintainer, head over to [the releases](https://github.com/encode/starlette/releases) section, and create a new draft release...

Screenshot 2021-11-17 at 15 52 25

Make sure to hit the "save draft" button not the "publish release".

Screenshot 2021-11-17 at 15 51 13

And pop back to this thread so we can approve it.

*Aside: It'd kinda be nice from my POV if GH had some more control over this process. Still find it a bit easy too easy get it unintentionally wrong.*

*I suppose we could **alternately** consider doing this process the other way around, and merging a release PR before we author the release stuff?*

**Comment by LarsStegman** [comment]
I have created the [draft release](https://github.com/encode/starlette/releases/tag/untagged-3c955988c265f4781516)

**Comment by lovelydinosaur** [comment]
@LarsStegman - Fab...

* Merge this PR.
* Publish the release.
* Check that [the publish action](https://github.com/encode/starlette/actions) runs okay, and that the new release [ends up on PyPI](https://pypi.org/project/starlette/).

**Comment by lovelydinosaur** [comment]
Hey presto! 🌟

It's not too bad our release process, eh? I'm sure we've got a few little improvements we could make to on boarding folks into maintainer-ship, and it'd be nice if this process had a few more guardrails perhaps, but pretty neat really, thanks to PyPI tokens & GitHub's actions support.

**Comment by LarsStegman** [comment]
Yeah, it's quite an easy process. It's nice there is no hassle to get the version deployed to PyPI.

**Pull request merged**